### PR TITLE
Update action.yaml zoom

### PR DESCRIPTION
**Pull request by raszpl**

### CODE REVIEW

File: render-pcb.sh [hunk 1]
```diff
@@ -25,13 +25,14 @@ extract_output_path() {
...
 
-while getopts :f:o:a:b:hv option
+while getopts :f:o:a:b:z:hv option
 do
     case "${option}" in
         f) kicad_pcb=${OPTARG};;
         o) output_path=${OPTARG};;
         a) animation=${OPTARG};;
         b) background=${OPTARG};;
+        b) zoom=${OPTARG};;
```
Comment (CRImier) [inline render-pcb.sh:35]: meant to do `z)`?

File: render-pcb.sh [hunk 2]
```diff
@@ -68,13 +69,13 @@ echo "$output_path"
 mkdir -p "$output_path"
 
 echo "rendering top"
-$KICAD_CLI pcb render --side top --background $background -o "$output_top" "$kicad_pcb"
+$KICAD_CLI pcb render --side top --background $background --zoom $zoom -o "$output_top" "$kicad_pcb"
```
Comment (CRImier) [inline render-pcb.sh:72]: does this still work if the zoom parameter is not present?

**Review by CRImier — COMMENTED** [review]
some fixes/tests appear to be needed

**Comment by raszpl** [comment]
thanks, yeah I banged this quickly while very frustrated, thus typo

**Comment by CRImier** [comment]
all understandable^^ push new commit into your branch and the PR should autoupdate. now realizing I should've proposed changes so that it's easier for you to accept but hey

File: render-pcb.sh (hunk 1 above)
Comment (CRImier) [inline render-pcb.sh:35]:
```suggestion
        z) zoom=${OPTARG};;
```

**Review by CRImier — CHANGES_REQUESTED** [review]

**Comment by linalinn** [comment]
Hi,
I am gonna try to merge it soon. May need to make changes even if it works currently e.g. if called with an empty `--zoom` which might change with newer versions.

**Pull request merged**